projects/packages/sync/src/modules/class-protect.php

```diff
@@ -46,8 +60,9 @@ public function init_listeners( $callback ) {
 	 * @param array $failed_attempt Failed attempt data.
 	 */
 	public function maybe_log_failed_login_attempt( $failed_attempt ) {
-		$brute_force_protection = Brute_Force_Protection::instance();
-		if ( $brute_force_protection->has_login_ability() && ! Jetpack_Constants::is_true( 'XMLRPC_REQUEST' ) ) {
+		$has_login_ability = apply_filters( 'jpp_has_login_ability', $this->has_login_ability_fallback() );
```

Comment:
Could you add a docblock for this new filter, so it can be parsed and a Codex page can be automatically created? This may be an opportunity to rename it to use the proper `jetpack_` prefix too?

Reply:
Thanks @jeherve, applied these adjustments via https://github.com/Automattic/jetpack/pull/38518/commits/d00182e90e0e021cd8333920ebd96c5b165ed6a8 :+1: